content/release-10-14-0/devices-10-14-0-bundle/10-14-0-204.md

```diff
@@ -0,0 +1,29 @@
+---
+weight: 32
+title: Release 10.14.0.204
+layout: bundle
+---
+
+<!--14.0.0.183 - 14.0.0.204-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>The data type mapping for the "Asset properties" widget has been improved to ignore null values for properties created by default when using MQTT templates, which prevented users from saving widget data unless the mentioned properties were fixed manually. Affected properties: c8y_Firmware.url, c8y_Software.url, c8y_Mobile.imei, c8y_Mobile.imsi, c8y_Mobile.cellId, c8y_Mobile.lac, c8y_Mobile.mcc, c8y_Mobile.mnc.</td>
```

Comment:
are the quotation marks the standard way to markup the widget name here?
same question for the properties list - they look like they could also be monospace font.

Reply:
all widget types are always formatted in quotes, see https://cumulocity.com/guides/users-guide/cockpit/#widgets-collection.
Regarding the properties, yes could be code format but not really required here, as this is not inside text and clear from the context. I try to keep the formatting simple in the release notes as we will not be able to do sophisticated formatting in the future with automation.